infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -135,7 +139,28 @@ export class AtatWebApiStack extends cdk.Stack {
     // TODO: getApplicationStep
     // TODO: createApplicationStep
     // TODO: submitPortfolioDraft
-    // TODO: uploadTaskOrder
+    addCreateTaskOrderFiles(this, taskOrderFiles);
     // TODO: deleteTaskOrder
   }
 }
+
+const addCreateTaskOrderFiles = (scope: cdk.Stack, resource: apigw.Resource) => {
```

Comment:
I really want to get away from this declaration syntax unless we have a good reason to use it.

```suggestion
function addCreateTaskOrderFiles(scope: cdk.Stack, resource: apigw.Resource) {
```

and then drop the semicolon at the end?

Reply:
Fair enough - probably good to avoid any confusion with usage of `this`. Will change.